packages/pigment-css-react/tests/theme/fixtures/themed-component.input.js

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { styled } from '@pigment-css/react';
+
+const StatRoot = styled('div', {
+  name: 'MuiStat', // The component name
```

Comment:
```suggestion
  name: 'PigmentStat', // The component name
```

Change it here and in all x other instances! :)